torch/csrc/stub.cpp

```diff
@@ -0,0 +1,18 @@
+#include <Python.h>
```

Comment:
I expect/intend for this stub file in the future, and just build _C.so directly with cmake. However for now, this stub file allows me to worry about getting all of the torch-python-in-cmake bits right, without having to deal with making sure that I've mimicked all the behaviors of setuptools, e.g. regarding where it puts its outputs